### PR TITLE
Use constants for custom workflow carrier account types

### PR DESCRIPTION
**Pull request by nwithan8**
# Description

- Internally, uses constants from checking custom workflow carrier account type names
- These constants are publicly available for end-users to use if they want
- BREAKING CHANGE: Path to `Constants` file has changed, if users were utilizing it for exception parsing

# Testing

- N/A

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: EasyPost/Constants.cs [hunk 1]
```diff
@@ -90,5 +90,17 @@ public static class ErrorMessages
             internal const string ApiErrorDetailsParsingError = "RESPONSE.PARSE_ERROR"; // not for public consumption
             public const string CouldNotPassClient = "Could not pass client to {0}.";
         }
+
+        public static class CarrierAccountTypes
```
Comment (Justintime50) [inline EasyPost/Constants.cs:94]: Question: Do we want to separate normal constants and error strings to separate files?

File: EasyPost/Constants.cs [hunk 2]
```diff
@@ -90,5 +90,17 @@ public static class ErrorMessages
...
+        {
+            public const string FedExAccountType = "FedexAccount";
+            public const string UpsAccountType = "UpsAccount";
+
+            internal static List<string> CustomWorkflowCarrierAccountTypes => new List<string>
+            {
+                FedExAccountType,
+                UpsAccountType
```
Comment (jchen293) [inline EasyPost/Constants.cs:102]: Nit: No need to define these variables, I would just pass them directly

**Review by jchen293 — APPROVED** [review]

File: EasyPost/Constants.cs (hunk 1 above)
Comment (nwithan8) [reply EasyPost/Constants.cs:94]: Having two constants files seems redundant and counter-intuitive to me.

File: EasyPost/Constants.cs (hunk 2 above)
Comment (nwithan8) [reply EasyPost/Constants.cs:102]: This way, users can use the constants if they'd like

Comment (nwithan8) [reply EasyPost/Constants.cs:102]: When building their data for the API call. That way they don't have to remember if it's "FedExAccount" or "FedexAccount" (it's the latter)
image

Comment (jchen293) [reply EasyPost/Constants.cs:102]: Ah, ok! Makes sense to me now

File: EasyPost/Constants.cs [hunk 3]
```diff
@@ -90,5 +90,17 @@ public static class ErrorMessages
...
+            };
+        }
```
Comment (Justintime50) [inline EasyPost/Constants.cs:104]: This is so much code where it's not necessary. KISS.

Let's have it be as simple as this psuedo code:

`CARRIER_TYPES_WITH_CUSTOM_WORKFLOWS = ['FedexAccount', 'UpsAccount']`

We need to be writing the minimal amount of code possible to get tests to pass/behavior met. The more code we have, the more liability we have. Let's design for what we need today (always) and refactor for what we need tomorrow when tomorrow comes because sometimes tomorrow doesn't even come. If ever there comes a day where there is another carrier with a custom workflow (unlikely), we'd just append a new string to the simple list.

TLDR: Let's just use a flat constant instead of a whole class for this.

**Review by Justintime50 — CHANGES_REQUESTED** [review]

File: EasyPost/Constants.cs (hunk 3 above)
Comment (nwithan8) [reply EasyPost/Constants.cs:104]: I agree. My one concern is, if someone were to send `type="FedExAccount"` (not FedEx vs Fedex), it wouldn't be captured by the list, and it would attempt to hit the normal endpoint with the data (granted, the data is bad anyway; assuming the API is case-sensitive for account type strings)

Comment (Justintime50) [reply EasyPost/Constants.cs:104]: Valid, though I thiiink the API would fail because we validate the type there so it may be a non-issue

**Review by Justintime50 — APPROVED** [review]
